src/main/java/HomePage.java

```diff
@@ -1,48 +1,75 @@
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
 
-public class HomePage extends BasePage{
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+public class HomePage extends BasePage {
+
+    private static final Logger log = LogManager.getLogger(HomePage.class);
```

Comment:
make it public and create metods info, error, debug where you could use in parameters String text.
your created log show at methods with parameters.

Reply:
1. You could try and check
2. Sorry didn't get your question